server/api.js

```diff
@@ -211,8 +211,8 @@ async function githubWebhook(req, res) {
 		await queuePush(fromPR(body));
 	}
 
-	// push to master
-	if (type === 'push' && body.ref === 'refs/heads/master' && body.head_commit) {
+	// push to trunk
+	if (type === 'push' && body.ref === 'refs/heads/trunk' && body.head_commit) {
```

Comment:
The `body.ref` value comes from the content of a GitHub webhook notification. It needs to remain `refs/heads/master` until the actual webhooks from GitHub change.